### ISSUE TITLE
RTIC support is broken

### ISSUE THREAD

**Issue by mciantyre** [imxrt-rs/imxrt-ral#14]
`imxrt-ral` 0.4 doesn't seem to support RTIC. See [this build](https://github.com/imxrt-rs/imxrt-ral/actions/runs/730646766) for an example of the failure.

**Comment by mciantyre** [comment]
Issue seems to be due to the duplicate `Peripherals` types that are available in the crate root. The module exports for a 1062 crate resemble

```rust
pub use cortex_m::armv7em::*;
pub use imxrt106::imxrt1062::*;
```

Both of these export a `Peripherals` type. RTIC can't figure out which one to use when using ```use imxrt_ral::Peripherals;```.

Will be solved by #8. Rebasing #13 onto that branch fixes the RTIC example.

**Comment by mciantyre** [comment]
I was wrong; we can support RTIC in the 0.4 release. Use the full path to the chip peripherals, instead of the crate root.

```rust
// Bad:
#[rtic::app(device = imxrt_ral, peripherals = true)]
mod app { /* ... */ }

// Good:
#[rtic::app(device = imxrt_ral::imxrt106::imxrt1062, peripherals = true)]
mod app { /* ... */ }
```

See [the teensy4 example](https://github.com/imxrt-rs/imxrt-ral/blob/26ffffc8990132119d54b9b6db74d76d4516ded4/examples/teensy4/rtic-blink.rs#L18) on the v0.4 branch for a demo. The next RAL release will let us use the crate root, too.
